pkg/pillar/types/zedroutertypes.go

```diff
@@ -2592,6 +2567,37 @@ func (config *NetworkInstanceConfig) IsIPv6() bool {
 	return false
 }
 
+// WithUplinkProbing returns true if the network instance is eligible
+// for uplink probing (see pkg/pillar/uplinkprober).
```

Comment:
Would it be useful to add a comment that we never bother probing when there is no choice of port?

Reply:
Added comment.